src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Main {
+
+    private static final Figure[] figures = new Figure[new Random().nextInt(3, 6)];

```

Comment:
we shouldn't create random in this class, all logic connected to generating should be encapsulated in figureSupplier